### PR TITLE
Chore: Update Learn Design systems to 6.0 

### PR DESCRIPTION
**Pull request by jonniebigodes**
With this pull request the Learn Design Systems tutorial was updated to 6.0.

What was done:
- Updated chapters to reflect the new changes with 6.0.
- The image/videos are matching the new version.
- Code snippets are updated to 6.0

What is not done:
- ~The architecture chapter is still using svn. I'm currently tracking how it was setup and see how to change it to degit and simplify it. @tmeasday i'll ping you in slack so that we can solve this and simplify it.~

The code used for the tutorial is hosted [here](https://github.com/jonniebigodes/intro-storybook-with-storybook-6-0https://github.com/jonniebigodes/intro-storybook-with-storybook-6-0)

Feel free to provide feedback

### CODE REVIEW

File: content/design-systems-for-developers/react/en/build.md [hunk 1]
```diff
@@ -138,92 +152,48 @@ export const buttonWrapper = () => (
 )
```
Comment (tmeasday) [inline content/design-systems-for-developers/react/en/build.md:152]: Can we fix this formatting?

Comment (tmeasday) [reply content/design-systems-for-developers/react/en/build.md:152]: Also I don't think we need `action` any more here if the stories use args.

File: content/design-systems-for-developers/react/en/build.md [hunk 2]
```diff
@@ -138,92 +152,48 @@ export const buttonWrapper = () => (
...
-    size={select('Size', ['tiny', 'small', 'medium', 'large'])}
-    username="Dominic Nguyen"
-    src="https://avatars2.githubusercontent.com/u/263385"
-  />
-);
-
-knobs.story = {
-  decorators: [withKnobs],
+// New story using controls
+export const controls = args => <Avatar {...args} />;
```
Comment (tmeasday) [inline content/design-systems-for-developers/react/en/build.md:178]:
```suggestion
const Template = args => <Avatar {...args} />;
export const Controls = Template.bind({});
```

File: content/design-systems-for-developers/react/en/document.md [hunk 3]
```diff
@@ -137,59 +113,53 @@ By default, every Avatar story is rendered in the docs. We can’t assume other
 ```javascript
 // src/Avatar.stories.js
 
-export const sizes = () => (
+export const sizes = args => (
```
Comment (tmeasday) [inline content/design-systems-for-developers/react/en/document.md:116]:
```suggestion
export const Sizes = args => (
```

File: content/design-systems-for-developers/react/en/workflow.md [hunk 4]
```diff
@@ -30,34 +30,73 @@ Download `AvatarList` component and story to your machine. Place it in the `/src
...
+![Storybook with AvatarList component](/design-systems-for-developers/storybook-with-avatarlist-6-0.png)
 
 Nice! Now let’s articulate each UI state supported by AvatarList. At a glance, it’s clear that AvatarList supports some of Avatar’s properties like `small` and `loading`.
 
 ```javascript
 // src/AvatarList.stories.js
 
-export const smallSize = () => <AvatarList users={users.slice(0, 2)} size="small" />;
-export const loading = () => <AvatarList loading />;
+export const smallSize = Template.bind({});
```
Comment (tmeasday) [inline content/design-systems-for-developers/react/en/workflow.md:40]:
```suggestion
export const SmallSize = Template.bind({});
```

**Review by tmeasday — CHANGES_REQUESTED** [review]
Great stuff! Looks great, so good to have this up to date.

The main thing is I think we use Capitalized names for stories, usually.

**Comment by jonniebigodes** [comment]
With the latest commit the architecture chapter is now updated to use degit instead of svn. And a bit more streamlined to avoid the user going back and forward to get their initial setup ready. Now he/she grabs the "template" runs yarn and he's good to go.

Feel free to provide feedback

File: content/design-systems-for-developers/react/en/build.md (hunk 2 above)
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/build.md]: fixed with the latest commit.

File: content/design-systems-for-developers/react/en/document.md (hunk 3 above)
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/document.md]: Fixed with the latest commit

File: content/design-systems-for-developers/react/en/workflow.md (hunk 4 above)
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/workflow.md]: Fixed with the latest commit

File: content/design-systems-for-developers/react/en/build.md (hunk 1 above)
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/build.md:152]: Formatting is fixed. Going to take a look at the action and if not necessary i'll update the snippet and commit it.

Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/build.md:151]: Just committed it. Forgot about that one. Thanks for it.

File: content/design-systems-for-developers/react/en/build.md [hunk 5]
```diff
@@ -101,129 +115,86 @@ The decorator will ensure the `GlobalStyle` is rendered no matter which story is
...
+  <source
+    src="/design-systems-for-developers/storybook-addon-controls-6-0.mp4"
+    type="video/mp4"
+  />
+</video>
 
-That said, knobs don’t replace stories. Knobs are great for exploring the edge cases of the components. Stories are used for showcasing the intended cases.
+That said, Controls don’t replace stories. They are great for exploring the edge cases of the components. Stories are used for showcasing the intended cases.
 
 We'll visit the Accessbility and Docs addons in later chapters.
```
Comment (domyen) [inline content/design-systems-for-developers/react/en/build.md:199]:
```suggestion
We'll visit the Accessibility and Docs addons in later chapters.
```

**Review by domyen — APPROVED** [review]
Looks good! Minor rfcs in comments.

File: content/design-systems-for-developers/react/en/architecture.md [hunk 6]
```diff
@@ -35,30 +35,45 @@ As such, our design system must be independent of any project and also a depende
...
-React is the most popular view layer according to the [State of JS](https://stateofjs.com/) survey. An overwhelming number of Storybooks use React, so we’re using it in this tutorial along with the popular [create-react-app](https://github.com/facebook/create-react-app) boilerplate.
+React is the most popular view layer according to the [State of JS](https://stateofjs.com/) survey. An overwhelming number of Storybooks use React, so we’re using it in this tutorial along with the custom [create-react-app](https://github.com/facebook/create-react-app) boilerplate.
 
-```bash
-npx create-react-app learnstorybook-design-system
+In a terminal run the following commands:
+
+```shell
+# Clone the files
+npx degit chromaui/learnstorybook-design-system#setup learnstorybook-design-system
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/architecture.md:44]: This is similar to the way svelte is setup with a small change.

What is happening here is the following, degit will get the files from the chromaui repo but from a branch, the setup branch. and place them inside the learnstorybook-design-system folder. 

With this we're shortening this section all together, by having the reader pull in the files necessary and no more delete src, svn the startup files and then the assets. 

In a single go he/she has the initial setup taken care of.

**Comment by domyen** [comment]
Nice! Sounds like a much better DX. 💪

File: content/design-systems-for-developers/react/en/architecture.md [hunk 7]
```diff
@@ -35,30 +35,45 @@ As such, our design system must be independent of any project and also a depende
 
 ## Setup our repository with create-react-app and GitHub
 
-React is the most popular view layer according to the [State of JS](https://stateofjs.com/) survey. An overwhelming number of Storybooks use React, so we’re using it in this tutorial along with the popular [create-react-app](https://github.com/facebook/create-react-app) boilerplate.
+React is the most popular view layer according to the [State of JS](https://stateofjs.com/) survey. An overwhelming number of Storybooks use React, so we’re using it in this tutorial along with the custom [create-react-app](https://github.com/facebook/create-react-app) boilerplate.
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/architecture.md]: Fixed with the latest commit. Marking as solved. Thanks for the feedback

File: content/design-systems-for-developers/react/en/architecture.md [hunk 8]
```diff
@@ -35,30 +35,45 @@ As such, our design system must be independent of any project and also a depende
...
 
-```bash
-npx create-react-app learnstorybook-design-system
+In a terminal run the following commands:
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/architecture.md]: Fixed with the latest commit. Marking as solved. Thanks for the feedback

File: content/design-systems-for-developers/react/en/build.md [hunk 9]
```diff
@@ -38,20 +38,27 @@ Storybook is the industry-standard [component explorer](https://blog.hichroma.co
...
-![Initial Storybook UI](/design-systems-for-developers/storybook-initial.png)
+![Initial Storybook UI](/design-systems-for-developers/storybook-initial-6-0.png)
 
 Nice, we’ve set up a component explorer!
 
-Your Storybook should reload like this (notice that the font styles are a little off, for instance see the "Initials" story):
+Every time you install Storybook into an application it will add some examples inside the `stories` folder. If you want, take some time and explore them. But we won't be needing them for our design system, so it's safe to delete the `stories` directory.
 
-![Initial set of stories](/design-systems-for-developers/storybook-initial-stories.png)
+Now your Storybook should look like this (notice that the font styles are a little off, for instance see the "Initials" story):
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/build.md]: Fixed with the latest commit. Marking as solved. Thanks for the feedback

File: content/design-systems-for-developers/react/en/build.md [hunk 10]
```diff
@@ -101,129 +115,86 @@ The decorator will ensure the `GlobalStyle` is rendered no matter which story is
...
+controls.args = {
+  loading: false,
+  size: 'tiny',
+  username: 'Dominic Nguyen',
+  src: 'https://avatars2.githubusercontent.com/u/263385',
 };
 ```
 
-Notice the Knobs tab in the addon panel. We instrumented the “Size” select element that allows us to cycle through the supported Avatar sizes `tiny`, `small`, `medium`, and `large`. You can instrument other props with knobs as well to create an interactive playground for the component.
+Notice the Controls tab in the addon panel. We instrumented the “size” select element that allows us to cycle through the supported Avatar sizes `tiny`, `small`, `medium`, and `large`. The same was applied to the remainder component's props (“loading”,“username” and “src”), allowing us to create a simple and user friendly way to interact and stress test components.
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/build.md]: Fixed with the latest commit. Marking as solved. Thanks for the feedback

File: content/design-systems-for-developers/react/en/distribute.md [hunk 11]
```diff
@@ -60,29 +60,46 @@ export * from './Icon';
 export * from './Link';
 ```
 
-Let’s add a development dependency on `@babel/cli` and `cross-env` to compile our JavaScript for release:
+We'll need some additional development packages, we're going to use [`@babel/cli`](https://www.npmjs.com/package/@babel/cli) and [`cross-env`](https://www.npmjs.com/package/cross-env) to help us with the build process.
 
-```bash
+In your terminal issue the following command:
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/distribute.md]: Fixed with the latest commit. Marking as solved. Thanks for the feedback

File: content/design-systems-for-developers/react/en/distribute.md [hunk 12]
```diff
@@ -91,12 +108,18 @@ dist
 
 #### Adding package metadata for publication
 
+To ensure consumers of the package get all the information necessary, some additional work is required on our `package.json`. The easiest way to to it, is simply running `yarn init` -- a command that initializes the package for publication:
+
+<!-- To ensure consumers of the package get all the information we want, we'll need to continue updating the `package.json`. The easiest way to do it is to run `yarn init` -- a command that initializes the package for publication: -->
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/distribute.md]: Fixed with the latest commit. Marking as solved. Thanks for the feedback

File: content/design-systems-for-developers/react/en/distribute.md [hunk 13]
```diff
@@ -332,77 +355,98 @@ Earlier in this tutorial, we standardized on a popular frontend stack that inclu
 
 The example app uses Storybook to facilitate [Component-Driven Development](https://www.componentdriven.org/), an app development methodology of building UIs from the bottom up starting with components and ending with pages. During the demo, we’ll run two Storybook’s side-by-side: one for our example app and one for our design system.
 
-Clone the example app repository from GitHub
+Run the following commands in your terminal to setup the example app:
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/distribute.md]: Fixed with the latest commit. Marking as solved. Thanks for the feedback

File: content/design-systems-for-developers/react/en/distribute.md [hunk 14]
```diff
@@ -332,77 +355,98 @@ Earlier in this tutorial, we standardized on a popular frontend stack that inclu
...
-
-Alternatively, you can browse your design system’s Storybook online if you deployed it to <a href="https://www.learnstorybook.com/design-systems-for-developers/react/en/review/#publish-storybook">Chromatic </a> earlier (see chapter 4).
+![Example app storybook with design system stories](/design-systems-for-developers/example-app-storybook-with-design-system-stories-6-0.png)
 
-We’ll use the Avatar component from our design system in the example app’s UserItem component. UserItem should render information about a user including a name and profile photo.
+We’ll use the `Avatar` component from our design system in the example app’s `UserItem` component. `UserItem` should render information about a user including a name and profile photo.
 
-Navigate to the UserItem.js component in your editor. Also, find UserItem in the Storybook sidebar to see code changes update instantly with hot module reload.
+In your editor, open the `UserItem` component, located in `src/components/UserItem.js`. Also, select `UserItem` in your Storybook, to see the code changes we're about to make instantly with hot module reload.
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/distribute.md]: In your editor, open the `UserItem` component located in `src/components/UserItem.js`. Also, select `UserItem` in your Storybook, to see the code changes we're about to make instantly with hot module reload.

File: content/design-systems-for-developers/react/en/document.md [hunk 15]
```diff
@@ -104,9 +80,9 @@ export function Avatar({ loading, username, src, size, ...props }) {
 
 You should now see this:
 
-![Storybook docs tab with component details](/design-systems-for-developers/storybook-docspage.png)
+![Storybook docs tab with component details](/design-systems-for-developers/storybook-docspage-6-0.png)
 
-Storybook Docs automatically generated the prop table that shows types and default values. That’s convenient, but it doesn’t mean Avatar is dummy-proof – several props can be misused. Add comments in your proptypes to render them in the auto-generated prop table.
+Storybook Docs automatically generated the args table that shows types and default values. That’s convenient, but it doesn’t mean Avatar is dummy-proof – several inputs (props) can be misused. Add comments in your proptypes to render them in the auto-generated args table.
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/document.md]: Fixed with the latest commit. Marking as solved. Thanks for the feedback

File: content/design-systems-for-developers/react/en/build.md (hunk 5 above)
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/build.md]: Fixed with the latest commit. Marking as solved. Thanks for the feedback

File: content/design-systems-for-developers/react/en/document.md [hunk 16]
```diff
@@ -199,18 +169,19 @@ Create a new `src/Avatar.stories.mdx` file and supply some details. We’ll remo
...
 
-<Meta title="Design System|Avatar" component={Avatar} />
+<Meta title="Design System/Avatar" component={Avatar} />
 
 # Avatar
 
 ## Displays an image that represents a user or organization
 
-Use an avatar for attributing actions or content to specific users. The user's name should _always_ be present when using Avatar – either printed beside the avatar or in a tooltip.
+Use an avatar for attributing actions or content to specific users.
```
[the remark this answers is not in the document]
Comment (jonniebigodes) [reply content/design-systems-for-developers/react/en/document.md:184]: prettier ignore should be applied to the snippet.

**Comment by jonniebigodes** [comment]
Merging it now.

**Pull request merged**